scipy/spatial/qhull.pyx

```diff
@@ -2470,7 +2487,7 @@ class Voronoi(_QhullUser):
            [ 0.5,  1.5],
            [ 1.5,  1.5]])
 
-    There is a single finite Voronoi region, and four finite Voronoi
+    There is a single finite Voronoi region, and eight infinite Voronoi
```

Comment:
Let me reply to the supposed docstring error in the morning.  I could very well be wrong.  Actually, I don't think so but there is no reason to be in such a hurry that it gets worse rather than better.

The "good" thing seems to be really solid.  I have been using it to play around with the inner/outer hull thing we talked about on a different thread (Spherical Voronoi diagrams with all the points in a single hemisphere) and it seems to be rock solid.  Since I am just using qhull this isn't a bit surprise.  I have not figured out how to do self tests in this framework, but I do have several tests that, when I figure it out, I will add to the pull request.

If you clone the repository I quoted above ( https://github.com/rtshort/spatial.git) I have a really cute 2d test program and a much less sophisticated 3d test program.